### DISCUSSION TITLE
not working move to dynamic route page with browser next page button

### DISCUSSION THREAD

**Discussion by JYPark88** [vercel/next.js#23806] [Help]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

10.15.3

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

This bug occurs when I move to dynamic route page with browser `next` button

error like below
Uncaught (in promise) Error: The provided `as` value (/mobile/order/23000098995081) is incompatible with the `href` value (/ile/order/[orderId]).

This is probably because I designated basePath in three letters, so It looks the three letters of router's aspath were cut off.

### Expected Behavior

Move to next page have to success.

### To Reproduce

1. set basePath.
2. move to the dynamic route page through `router.push`
3. move to previous page with browser button.
4. move to next page with browser button.

**Comment by JYPark88** [comment]
Can I know why this issue closed and moved to discussion?

**Reply by colinhemphill** [reply]
Yeah, because I'm seeing the same issue. We have an application using `basePath` and after a recent Next upgrade we started seeing `The provided as value is incompatible with the href value` in Sentry. Our application does not use `as=""` explicitly in _any_ links, so the values that we're using for dynamic routing are not causing this.

Like @JYPark88 said, it only happens with the browser navigation. Buttons with navigation links on them are working fine for me.

**Reply by m235** [reply]
We're experiencing the same thing. Hope there will be a fix soon

**Reply by colinhemphill** [reply]
I believe this issue is related to the same discussion, so might be a good idea to follow up there so that it gets some attention. I haven't had a chance to dig into source to see if I can come up with a fix for it.

https://github.com/vercel/next.js/issues/23809
